packages/repository-tests/src/crud/relations/acceptance/has-many-inclusion-resolver.relation.acceptance.ts

```diff
@@ -0,0 +1,273 @@
+            expect(customerRepo.inclusionResolvers).to.deepEqual(new Map());
+
+            await ctx.dataSource.automigrate([Customer.name, Order.name]);
+          }),
+        );
+
+        beforeEach(async () => {
+          customerRepo.inclusionResolvers.set(
+            'orders',
+            customerRepo.orders.inclusionResolver,
+          );
+          customerRepo.inclusionResolvers.set(
+            'customers',
+            customerRepo.customers.inclusionResolver,
+          );
+          await customerRepo.deleteAll();
+          await orderRepo.deleteAll();
+        });
+
+        it("defines a repository's inclusionResolvers property", () => {
+          expect(customerRepo.inclusionResolvers).to.not.be.undefined();
+          expect(orderRepo.inclusionResolvers).to.not.be.undefined();
+        });
+
+        it("throws an error if the repository doesn't have such relation names", async () => {
+          await orderRepo.create({
+            customerId: existingCustomerId,
+            description: 'Order from Order McForder, the hoarder of Mordor',
+          });
+          await expect(
+            customerRepo.find({include: [{relation: 'managers'}]}),
+          ).to.be.rejectedWith(
+            `Invalid "filter.include" entries: {"relation":"managers"}`,
+          );
+        });
+
+        it('throws error if the target repository does not have the registered resolver', async () => {
+          await orderRepo.create({
+            customerId: existingCustomerId,
+            description: 'Order from Order McForder, the hoarder of Mordor',
+          });
+          // unregister the resolver
+          customerRepo.inclusionResolvers.delete('orders');
+
+          await expect(
+            customerRepo.find({include: [{relation: 'orders'}]}),
+          ).to.be.rejectedWith(
+            `Invalid "filter.include" entries: {"relation":"orders"}`,
+          );
+          // reset
+          customerRepo.inclusionResolvers.set(
+            'orders',
+            customerRepo.orders.inclusionResolver,
+          );
+        });
+
+        it('simple has-many relation retrieve via find() method', async () => {
+          const c1 = await customerRepo.create({name: 'c1'});
+          const o1 = await orderRepo.create({
+            customerId: c1.id,
+            description: 'order from c1',
+          });
+          const result = await customerRepo.find({
+            include: [{relation: 'orders'}],
+          });
+
+          const expected = {
+            id: c1.id,
+            name: 'c1',
+            orders: [
+              {
+                id: o1.id,
+                description: 'order from c1',
+                customerId: c1.id,
+                isShipped: features.emptyValue,
+                // eslint-disable-next-line @typescript-eslint/camelcase
+                shipment_id: features.emptyValue,
+              },
+            ],
+            parentId: features.emptyValue,
+          };
```

Comment:
Can we build the expected object using `c1` and `o1` instances? Something along the following lines:

```ts
const expected = {
  ...toJSON(c1), 
  orders: [toJSON(o1)],
};
```